### DISCUSSION TITLE
기본 텍스트를 전체보기 텍스트로 전달하는방법

### DISCUSSION THREAD

**Discussion by minmoong** [storycraft/node-kakao#547] [Q&A]
기본 텍스트를 전체보기 텍스트로 전달하는방법으로써 KnownChatType의 메시지 타입중 전체보기 메시지 타입이 존재하지 않습니다. 어떻게 기본 텍스트를 전체보기 텍스트로 전달합니까?

**Comment by kaan03** [comment]
카카오톡 메시지는 500글자가 넘을 시 전체보기화가 됩니다. 따라서 아래와 같이 공백글자 500개를 넣어 전체보기화를 시킬 수 있습니다.
```
const Lw = '\u200b'.repeat(500);
channel.sendChat(`전체보기: ${Lw}전체보기 후 표시될 메시지`);
```

**Reply by minmoong** [reply]
정말 감사합니다!

**Reply by tadashi74** [reply]
> 이렇게 할 경우 메시지양이 많을 때 뒤에 짤리던데 안짤리는 방법이 있을까요?

해당 문제는 카카오 글자수제한때문에 발생하는 문제로, 커스텀(?)전체보기를 포기하시는 방법밖에 없습니다

**Reply by secneuqes** [reply]
혹시 어떤 방법을 이용하셨나요?

[a comment by an automated account is not shown]
